$OpenBSD: patch-Makefile_PL,v 1.1 2011/04/09 08:11:43 jasper Exp $

Don't try to fetch missing dependencies from CPAN,
we take care of them with REGRESS_DEPENDS.

--- Makefile.PL.orig	Sat Apr  9 00:42:31 2011
+++ Makefile.PL	Sat Apr  9 00:42:54 2011
@@ -6,6 +6,7 @@ use warnings;
 name 'Parse-Method-Signatures';
 all_from 'lib/Parse/Method/Signatures.pm';
 
+=pod
 requires 'List::MoreUtils' => '0.20';
 requires 'Moose';
 requires 'MooseX::Traits' => '0.06';
@@ -20,6 +21,7 @@ test_requires 'Test::More';
 test_requires 'Test::Exception';
 test_requires 'Test::Differences';
 test_requires 'MooseX::Types::Structured';
+=cut
 
 postamble(<<"EOM");
 testcover: pm_to_blib
@@ -34,5 +36,5 @@ resources(
 );
 
 auto_provides;
-auto_install();
+#auto_install();
 WriteAll;
